docs/guides/storage/examples_storagehelper.md

```diff
@@ -65,10 +67,48 @@ By default, the `StorageManager` reports its upload progress to the console ever
 [`StorageManager.set_report_upload_chunk_size`](../../references/sdk/storage.md#storagemanagerset_report_upload_chunk_size) 
 class method, and specifying the chunk size in MB (not supported for Azure and GCP storage).
 
+```python
+StorageManager.set_report_upload_chunk_size(chunk_size_mb=10)
+```
+
+## Working with Folders
+### Downloading a Folder
+Download a folder to a local machine using the [`StorageManager.download_folder`](../../references/sdk/storage.md#storagemanagerdownload_folder) 
+class method. Specify the remote storage location as the `remote_url` argument and the target local location as the 
+`local_folder` argument. 
+
+```python
+StorageManager.download_folder(remote_url="s3://bucket/", local_file="/folder/’")
+```
+
+This method downloads a remote folder recursively, maintaining the sub-folder structure from 
+the remote storage.
+
+For example: if you have a remote file `s3://bucket/sub/file.ext`, then 
+`StorageManager.download_folder(remote_url="s3://bucket/", local_file="/folder/")` will create `/folder/sub/file.ext`.
+
+You can input `match_wildcard` so only files matching the wild card are downloaded.
+
+### Uploading a Folder
+Upload a local folder to remote storage using the [`StorageManager.upload_folder`](../../references/sdk/storage.md#storagemanagerupload_folder) 
+class method. Specify the local folder to upload as the `local_folder` argument and the target remote location as the
+`remote_url` argument. This method uploads the local folder recursively to remote storage, maintaining the sub-folder structure from the local 
+storage. 
+
+```python
+StorageManager.upload_folder(local_file="/LocalFolder", remote_url="s3://MyBucket/MyFolder")
+```
+
+For example: If you a local file `/LocalFolder/sub/file.ext` then `StorageManager.upload_folder(local_file="/LocalFolder", remote_url="s3://MyBucket/MyFolder")`
+will create `s3://bucket/sub/file.ext`.
+
+Use the `retries` parameter to set the number of times folder upload should be retried in case of failure.
```

Comment:
@pollfly Is `retries` for the entire folder, or per file? What happens on failure? Is the partially uploaded folder kept or removed?